op-scim-bridge/chart/op-scim-bridge/Chart.yaml

```diff
@@ -1,5 +1,6 @@
-apiVersion: v1
```

Comment:
Did you have any luck leveraging our existing Helm Chart into this repo? 

See: https://gitlab.1password.io/dev/b5/op/-/issues/866

Reply:
Like you posted there, the `do-block-storage` option will need to be worked around in terms of the DO/GCP split. We can likely override it in some way when packaging it, but that'll have to be a larger conversation.